Resources/Private/Templates/Authentication/ShowLoginForm.html

```diff
@@ -3,7 +3,7 @@
 <f:layout name="Default"/>
 <f:section name="main">
 
-<f:flashMessages renderMode="div"/>
+<f:flashMessages />
```

Comment:
This will cause a [different behavior in TYPO3 >= v7](https://docs.typo3.org/typo3cms/extensions/core/7.6/singlehtml/#breaking-63453-changed-rendering-of-flashmessagesviewhelper).